src/cortexpy/tesserae.py

```diff
@@ -158,8 +155,8 @@ def __align_all(self, panel):
         size_l -= self.qlen
         lsize_l = np.log(size_l)
 
-        max_r, pos_max, state_max, who_max = self.__initialization(query, panel, lsize_l)
-        max_r, pos_max, state_max, who_max = self.__recurrence(query, panel, lsize_l, l1, max_r, pos_max, state_max, who_max, store_states=self.mem_limit)
+        max_r, pos_max, state_max, who_max = self.__initialization(query, targets, lsize_l)
+        max_r, pos_max, state_max, who_max = self.__recurrence(query, targets, lsize_l, l1, max_r, pos_max, state_max, who_max, store_states=self.mem_limit)
```

Comment:
I'm surprised this line passed flake8 testing on Travis-CI. There's supposed to be a 100 character line limit enforced by tox 😕